packages/backend/src/core/totalSupply/TotalSupplyTokensConfig.ts

```diff
@@ -0,0 +1,8 @@
+import { AssetId, EthereumAddress, UnixTime } from '@l2beat/shared-pure'
+
+export interface TotalSupplyTokensConfig {
+  assetId: AssetId
+  tokenAddress: EthereumAddress
+  sinceTimestamp: UnixTime
```

Comment:
Check whether it is still valid
cc @antooni

Reply:
it looks okay for now